lib/passport-configurator.js

```diff
@@ -294,31 +294,32 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
                     function(err, accessToken) {
                       if (err) {
                         return err.code === 'LOGIN_FAILED'
-                            ? done(null, false, {message: 'Login failed'})
+                            ? done(null, false, {message: errorMsg})
```

Comment:
Is there a reason to make the error messages generic? The ones already in place seem to identify the problem more specifically.